tests/Install-DbaSqlWatch.Tests.ps1

```diff
@@ -0,0 +1,58 @@
+$CommandName = $MyInvocation.MyCommand.Name.Replace(".Tests.ps1", "")
+Write-Host -Object "Running $PSCommandpath" -ForegroundColor Cyan
+. "$PSScriptRoot\constants.ps1"
+
+Describe "$CommandName Unit Tests" -Tag 'UnitTests' {
+    Context "Validate parameters" {
+        $paramCount = 6
+        $commonParamCount = ([System.Management.Automation.PSCmdlet]::CommonParameters).Count + 2
+        [object[]]$params = (Get-ChildItem function:\Install-DbaSqlWatch).Parameters.Keys
+        $knownParameters = 'SqlInstance', 'SqlCredential', 'Database', 'Force', 'LocalFile', 'EnableException'
+        It "Should contain our specific parameters" {
+            ( (Compare-Object -ReferenceObject $knownParameters -DifferenceObject $params -IncludeEqual | Where-Object SideIndicator -eq "==").Count ) | Should Be $paramCount
+        }
+        It "Should only contain $paramCount parameters" {
+            $params.Count - $commonParamCount | Should Be $paramCount
+        }
+    }
+}
+
+Describe "$CommandName Integration Tests" -Tags "IntegrationTests" {
+    Context "Testing SqlWatch installer" {
+        BeforeAll {
+            $database = "dbatoolsci_sqlwatch_$(Get-Random)"
+        }
+        AfterAll {
+            Remove-DbaDatabase -SqlInstance $script:instance2 -Database $database -Confirm:$false
+            Get-DbaAgentJob -SqlInstance $script:instance2 | Where-Object {$PSItem.Name -like "DBA-PERF-*" } | Remove-DbaAgentJob
+        }
+
+        $results = Install-DbaSqlWatch -SqlInstance $script:instance2 -Database $database -Force
```

Comment:
I didn't get any output at all in my tests - unsure what's up. Did it work for you? Please check out dbatools.io/ci to see the test failures.

Reply:
This one is a curious problem. I _was_ getting output on the unit tests when run locally, but now I'm not getting any output at all. I've got constants.local.ps1 set up to point `$script:instance2` to a local SQL 2016 box so I'll be able to debug the integration tests as well. 

I'll attempt to get this resolved tomorrow.

Reply:
Thanks so much! It's been strange for me too.